frontend/test/metabase/scenarios/onboarding/reference/reproductions/5276-remove-field-type.cy.spec.js

```diff
@@ -17,7 +17,11 @@ describe("issue 5276", () => {
     cy.findByText("Edit").click();
 
     cy.findByText("Score").click();
-    popover().within(() => cy.findByText("No field type").click());
+    popover().within(() => {
+      cy.findByText("Score").should("be.visible");
+      cy.get(".ReactVirtualized__Grid.MB-Select").scrollTo("bottom");
+      cy.findByText("No field type").click();
+    });
```

Comment:
@ranquild Since you created this test. Could you help take a look if the change is okay? Basically, I need to change the test because of how `TippyPopoverWithTrigger` is rendering `react-virtualized` differently.

Previously, we don't need to scroll to the bottom because somehow `No field type` which is the last option exists in the DOM.
![Screen Shot 2022-05-11 at 2 45 14 PM](https://user-images.githubusercontent.com/1937582/167796988-8c84fc13-1896-4a17-b10e-a145c65303fb.png)

But you could see that after migrating the `<Select />` component to `TippyPopoverWithTrigger` the last component rendered in the virtualized list is something different (it isn't the last option)
![Screen Shot 2022-05-11 at 2 46 22 PM](https://user-images.githubusercontent.com/1937582/167797103-5ff1bea5-997c-4f21-98ae-31aa86a592ff.png)

That's why I need to scroll to the bottom, but I need to wait for the `Score` text to appear first as it seem it take a few ticks for the virtualized list to scroll `Score` (selected option) into view.

Without line 21 the list won't scroll to the bottom. Or rather it's already scrolled to the bottom but `Score` will scroll into view afterward causing `No field type` to detach from the DOM.

Reply:
Would it be possible to rewrite this piece in a way that we don't reference the CSS class? E.g. we could search for the new field type

Reply:
I believed not. Because of `react-virtualized`, anything that we don't see is detached from the DOM, so we can only rely on what's visible on the screen. And I need to get the exact element in the DOM otherwise, I couldn't make the virtualized list scroll and there're 2 other places that do this. So, that's why I went with targeting CSS.

The 2 other places that do similar CSS lookup and scroll the virtualized container.

1. https://github.com/metabase/metabase/blob/0e145152b070693acc688282931a405bddc5be4a/frontend/test/metabase/scenarios/admin/datamodel/field-type.cy.spec.js#L86-L88
2. https://github.com/metabase/metabase/blob/fa90a66c76dcbde2ba5e7570344eb4b3b9f6cd50/frontend/test/metabase/scenarios/models/helpers/e2e-models-metadata-helpers.js#L15

For what it's worth here's how the scrolling div looks like
```html
<div aria-label="grid" aria-readonly="true" class="ReactVirtualized__Grid ReactVirtualized__List MB-Select text-brand" role="grid" tabindex="0" style="...">
```

And I could change line 22 to
```diff
-      cy.get(".ReactVirtualized__Grid.MB-Select").scrollTo("bottom");
+      cy.findByRole("grid").scrollTo("bottom");
```

But not sure if that's better or worse. What do you think?

Reply:
Looks better to me. We could also move it to a (shared) helper function if it's duplicated